### PR TITLE
use of fuzzy_bool instead of ternary_sort 

### LINKED ISSUE #13699: use fuzzy_bool instead of ternary_sort
There is an existing function that can take the place of `ternary_sort`:
```diff
diff --git a/sympy/sets/sets.py b/sympy/sets/sets.py
index 659b778..f050f12 100644
--- a/sympy/sets/sets.py
+++ b/sympy/sets/sets.py
@@ -8,6 +8,7 @@
 from sympy.core.expr import Expr
 from sympy.core.singleton import Singleton, S
 from sympy.core.evalf import EvalfMixin
+from sympy.core.logic import fuzzy_bool
 from sympy.core.numbers import Float
 from sympy.core.compatibility import (iterable, with_metaclass,
     ordered, range, PY3)
@@ -217,13 +218,8 @@ def _complement(self, other):
             return S.EmptySet

         elif isinstance(other, FiniteSet):
-            from sympy.utilities.iterables import sift
-
-            def ternary_sift(el):
-                contains = self.contains(el)
-                return contains if contains in [True, False] else None
-
-            sifted = sift(other, ternary_sift)
+            sifted = sift(other,
+                lambda x: fuzzy_bool(self.contains(x)))
             # ignore those that are contained in self
             return Union(FiniteSet(*(sifted[False])),
                 Complement(FiniteSet(*(sifted[None])), self, evaluate=False)
```

### PR DESCRIPTION
**Pull request by shubham3174**
changed ternary_sort with predefined fuzzy_bool

Fixes #13699

### CODE REVIEW

**Comment by gxyd** [comment]
You also created #13722 , any reason for two PR's?

**Comment by shubham3174** [comment]
It was showing that the Travis ci build failed, I thought the reason was that I had created a branch from a branch which was not master, so I created one more branch but this time from the master branch,  hence the two pr. Sorry though, I am new to git and this is my first submission in sympy

**Comment by gxyd** [comment]
That's okay. You should probably close one of them. This one?
Let's us know if you have any trouble using git.

File: sympy/geometry/tests/test_line.py [hunk 1]
```diff
@@ -699,3 +699,14 @@ def _check():
     # midline intersection
     c, d = (-2, -3), (-2, 0)
     _check()
+
+
+def tes_issue_13705
+x = Symbol('x')
```
Comment (smichr) [inline sympy/geometry/tests/test_line.py]: * the file needs to end with a newline
* the lines of the function need to be indented
* see this file (or others) for an example where `raises(ValueError,...)` is used..

File: sympy/physics/tests/test_secondquant.py [hunk 2]
```diff
@@ -1219,3 +1219,17 @@ def test_internal_external_pqrs_AT():
     ]
     for permut in exprs[1:]:
         assert substitute_dummies(exprs[0]) == substitute_dummies(permut)
+
+
+def test_13707():
+v = symbols("v")
```
Comment (smichr) [inline sympy/physics/tests/test_secondquant.py]: see comments above regarding indentation

File: sympy/sets/sets.py [hunk 3]
```diff
@@ -218,12 +218,8 @@ def _complement(self, other):
 
         elif isinstance(other, FiniteSet):
             from sympy.utilities.iterables import sift
-
-            def ternary_sift(el):
-                contains = self.contains(el)
```
Comment (smichr) [inline sympy/sets/sets.py]: leave ternary_sift in place but make it's 1 line contents be `return fuzzy_bool(self.contains(e1))`

**Comment by gxyd** [comment]
@smichr I am really sorry, I should have taken care of this. But right now there already is a PR https://github.com/sympy/sympy/pull/13703 in review for the related issue.

I'm closing this one. @shubham3174 I'm sorry for your waisted effort, but please ask on issue if you are unsure if someone else is working on it.

**Pull request closed without merging**